README.md

```diff
@@ -25,7 +25,7 @@ For more information, check out the [user guide][user-guide] or [FAQ][faq].
 
 1. Pytype uses **inference** instead of gradual typing. This means it will
 infer types on code even when the code has no type hints on it. So it can
-detect issues with code like this, which other type checkers would miss:
+detect issues with code which other type checkers would miss like this:
```

Comment:
I prefer the original structure here, for the reason that it keeps "like this" close to the word ("code") that the phrase describes.

Reply:
I see your point and will revert this one back